bindings.cpp

```diff
@@ -1900,6 +1900,7 @@ bool DeRestPluginPrivate::sendConfigureReportingRequest(BindingTask &bt)
         rq.maxInterval = 300;
         if (modelId == QLatin1String("SmartPlug") ||               // Heiman
             modelId == QLatin1String("SKHMP30-I1") ||              // GS smart plug
+            modelId == QLatin1String("SP2610ZB") ||                // Sinope smart in-wall outlet
```

Comment:
Based on your screenshot, I doubt this is the correct reportable change, which should reflect the value of 1 Wh. On the other hand, multiplier and divisor for the value are both 1, which would conclude a value of 56636 kWh going purely by the math, however, attribute 0x0303 is also not yet taken into account (which basically tells you where to set the decimal point).

Reply:
The value being stored in the Z2610ZB in this field is in Wh, even though it is configured as if it is in kWh.  The value being sent seems to be correct.